src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -86,10 +85,26 @@ export class OverflowMenuPane extends Dialog {
 				event.preventDefault();
 				listItems[listItems.length - 1].focus();
 				break;
+
+			case "Esc": // IE specific value
+			case "Escape": {
+				event.stopImmediatePropagation();
+				this.doClose();
+				break;
+			}
+		}
+	}
+
+	clickClose(event) {
+		if (this.dialogConfig.parentRef.nativeElement.firstChild.contains(event.target) ||
```

Comment:
Add a comment that in simple words explains in which cases to we not actually close.